### ISSUE TITLE
A brief lesson on copyright?

### ISSUE THREAD

**Issue by amlalejini** [amlalejini/2019-msu-pygame#1]
Alexa suggested we talk to students about copyright infringement. This is particularly relevant given the urge to just grab art/assets off other websites/google images.

**Comment by mmore500** [comment]
Good idea. We should also give links to resources with freely-licensed assets and show how to use the license filter on Google images.

**Comment by amlalejini** [comment]
For sure for sure. Maybe I can mine some links from Alexa.
